src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -22,6 +21,28 @@
 log = logging.getLogger(__name__)
 
 
+class SlackRetryException(Exception):
+    def __init__(self, wait_time: int | None = None):
+        self.wait_time = wait_time
+        if wait_time:
+            super().__init__(f"Retrying slack call in {wait_time} seconds.")
+        else:
+            super().__init__("Retrying slack call.")
+
+    def get_wait_time(self) -> int:
+        return self.wait_time
+
+def slack_wait_strategy(retry_state: RetryCallState) -> float:
```

Comment:
```suggestion
def slack_wait_strategy(retry_state: RetryCallState) -> int:
```

Reply:
Just do [float, int] I guess wait_exponential returns a float

Reply:
I leave it for @kevgliss to decide what is best there.